clinica/pipelines/statistics_volume/statistics_volume_utils.py

```diff
@@ -1,14 +1,27 @@
-def get_group_1_and_2(tsv, contrast):
-    """
-        Based on the TSV file given in parameter, compute indexes of each group
-    Args:
-        tsv: (str) path to the tsv file containing information on subjects/sessions with all covariates
-        contrast: (str) name of a column of the tsv
-
-    Returns:
-        first_group_idx: (list of int) list of indexes of first group
-        second_group_idx: (list of int) list of indexes of second group
-        class_names: (list of str of len 2) list of the class names read in the column contrast of the tsv
+import typing as ty
+from pathlib import Path
+
+
+def get_group_1_and_2(
+    tsv: str, contrast: str
+) -> ty.Tuple[ty.List[int], ty.List[int], ty.List[str]]:
```

Comment:
I'm getting a RunTimeError when running the Nipype workflow.
It's the good old issue of self-contained functions again... 
You'll need to undo this unfortunately...

```suggestion
) -> tuple:
```